extensions/amp-access/0.1/amp-access-server-jwt.js

```diff
@@ -121,10 +121,17 @@ export class AccessServerJwtAdapter {
     /** @private @const {string} */
     this.serviceUrl_ = serviceUrlOverride || removeFragment(win.location.href);
 
-    /** @const @private {string} */
-    this.keyUrl_ = user().assert(configJson['publicKeyUrl'],
-        '"publicKeyUrl" URL must be specified');
-    assertHttpsUrl(this.keyUrl_, '"publicKeyUrl"');
+    /** @const @private {?string} */
+    this.key_ = configJson['publicKey'] || null;
+
+    /** @const @private {?string} */
+    this.keyUrl_ = configJson['publicKeyUrl'] || null;
+
```

Comment:
Looks like if both are specified, url one will be ignored, maybe a `dev().warn` for that here?